site/app/controllers/forum/ForumController.php

```diff
@@ -594,58 +569,61 @@ private function getSortedThreads($categories_ids, $max_thread, $show_deleted =
     }
 
     public function getThreads(){
-
+        $pageNumber = !empty($_GET["page_number"]) && is_numeric($_GET["page_number"]) ? (int)$_GET["page_number"] : -1;
```

Comment:
We should be validating the lower bound of this page number, i.e. what if the page number is input as -15 (this can be done by simply changing the `next_page` html attribute). This will throw an exception since the offset in the query can't be negative.